src/pages/scheduler/scheduler.tsx

```diff
@@ -1,10 +1,14 @@
 import { Flex, Heading } from '@chakra-ui/layout';
+import { Helmet } from 'react-helmet';
 
 import { Header } from '../../app';
 
 export function Scheduler(): JSX.Element {
   return (
     <Flex h="100vh" direction="column">
+      <Helmet>
+        <title>Schedule Courses · clockwork</title>
+      </Helmet>
```

Comment:
I would suggest we have a top-level Helmet in `index.tsx` so we can use the `titleTemplate` and `defaultTitle`.

Reply:
Thanks for bringing that up, was unaware